content/admin/Advanced Configuration Options.adoc

```diff
@@ -0,0 +1,170 @@
+---
+Title: Advanced Configuration Options
+Weight: 52
+---
+
+
+:toc:
+
+:imagesdir: ./../../images/en/user
+
+
+= Advanced Configuration Options
+
+A system administrator who has access to the files installed by SuiteCRM has additional system configuration options described in this section.
+
+== Blocking Update Wizard and Module Builder, as well as blocking the creation of backup files of the system.
+
+If you, as a system administrator, manage multiple copies of SuiteCRM,
+then you can retain full control over the systems by prohibiting all users
+with administrative rights from making changes to the system.
+(It should be noted here that one should not confuse the system administrator who
+has access to the database and to the system files, and the SuiteCRM user with
+administrative rights). To do this, you can block user access to the
+Update Wizard, the Module Builder, and also prohibit creating backup copies of system files.
+
+For this in the file `config.php` set the following parameter:
+
+[source,php]
+['admin_access_control'] => true
+
+== Module Load Control
+
+To ensure that users of the system have administrative rights
+(but not having access to the configuration file `config.php`)
+cannot load extraneous modules into the system, you can limit
+the file downloads to the frames of a specific folder you control.
+This will allow you to control all modules loaded into the system.
+
+
+To control module loading, do the following :
+
+In the file `config.php` set the following parameters :
+
+[source,php]
+'use_common_ml_dir'=>'true',
+'common_ml_dir'=>'ml_directory',
+
+Where *ml_directory*  - the name of the folder from which the modules will be loaded.
+
+== Limiting system resources
+
+By default, any database calls requiring more than 1000 queries are terminated.
+For import, export, synchronization, and when working in the administration panel,
+the maximum number of requests is 50,000. Thus, all processes that take up too
+much system resources are interrupted. You can change these settings as
+desired by specifying values ​​for variables
+`defaultpass:[_]limit` and `specialpass:[_]querypass:[_]limit` in file `config.php`.
+
+== Setting file system permissions in Linux
+
+If the system is running on a computer running Linux,
+you can change the owner of the system files, as well as
+control access to all files and folders on the system, by setting user and group rights.
+
+The following example shows the read, write and execute permissions for the user and group Apache:
+
+[source,php]
+'default_permissions' => array(
+'dir_mode' => 02770,
+'file_mode' => 0660,
+'chown' => 'apache',
+'chgrp' => 'apache',),
+
+For the parameter `dirpass:[_]mode` you can set the value to 1528,
```

Comment:
I'm not sure, but maybe it would look good to enclose the numbers in backticks, so they get slightly different formatting. Choose whatever you think looks better :-)